Tools/scripts/generate_global_objects.py

```diff
@@ -14,6 +14,7 @@
     'DUNDER',  # Objects/typeobject.c
     'RDUNDER',  # Objects/typeobject.c
     'SPECIAL',  # Objects/weakrefobject.c
+    'NAME',  # Objects/typeobject.c
 }
 IDENTIFIERS = [
```

Comment:
I count 77 unique "dunder" names in typeobject.c but only 48 here in generate_global_objects.py.  Presumably the missing 29 identifiers okay at the moment due to direct use of `_Py_ID()` for the respective strings in other files.  However, they should also be here in `IDENTIFIERS` to ensure there isn't a problem if other code stops declaring the identifier independently.

<details>
<summary>FTR, the full list:</summary>

`grep -o 'SLOT("__[^"]*__"' Objects/typeobject.c | sort | uniq`

```
__abs__
__add__
__aiter__
__and__
__anext__
__await__
__bool__
__call__
__contains__
__del__
__delattr__
__delete__
__delitem__
__eq__
__float__
__floordiv__
__ge__
__get__
__getattr__
__getattribute__
__getitem__
__gt__
__hash__
__iadd__
__iand__
__ifloordiv__
__ilshift__
__imatmul__
__imod__
__imul__
__index__
__init__
__int__
__invert__
__ior__
__ipow__
__irshift__
__isub__
__iter__
__itruediv__
__ixor__
__le__
__len__
__lshift__
__lt__
__matmul__
__mod__
__mul__
__ne__
__neg__
__new__
__next__
__or__
__pos__
__pow__
__radd__
__rand__
__repr__
__rfloordiv__
__rlshift__
__rmatmul__
__rmod__
__rmul__
__ror__
__rpow__
__rrshift__
__rshift__
__rsub__
__rtruediv__
__rxor__
__set__
__setattr__
__setitem__
__str__
__sub__
__truediv__
__xor__
```
</details>

Reply:
Good catch I added the rest with your helper `grep` and `__divmod__` and `__rdivmod__` which were not included in your list.

Reply:
Awesome!